_i18n/de/resources/moneropedia/ringCT.md

```diff
@@ -1,14 +1,14 @@
 ---
-terms: ["ringCT", "ring-CT"]
-summary: "a way to hide the amount sent in a Monero transaction"
+terms: ["ringCT", "ring-CT", "Ring-CT", "Ring-CTs"]
+summary: "Eine Möglichkeit, den in Monero-Transaktionen gesendeten Betrag zu verstecken"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
-RingCT, short for Ring Confidential Transactions, is how transaction amounts are hidden in Monero.
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
+Ring-CTs, kurz für "Ring Confidential Transactions" (vertrauliche Ringtransaktionen), ermöglichen die Verschleierung von Transaktionsbeträgen bei Monero.
 
-Ring CT was implemented in block #1220516 in January 2017.  After September 2017, this feature became mandatory for all transactions on the network.
+Ring-CT wurde im Januar 2017 im Block #1220516 implementiert. Ab September 2017 wurde diese Funktion für alle Transaktionen innerhalb des Netzwerks obligatorisch.
 
-RingCT introduces an improved version of @ring-signatures called "A Multi-layered Linkable Spontaneous Anonymous Group signature", which allows for hidden amounts, origins and destinations of transactions with reasonable efficiency and verifiable, trustless coin generation.
+Ring-CT führt eine verbesserte Version von @Ringsignaturen - übersetzt "Eine mehrschichtige, verlinkbare, spontane, anonyme Gruppensignatur" genannt - ein. Diese berücksichtigt verborgene Beträge und Transaktionsursprünge und -ziele mit akzeptabler Effizienz und verifizierbarer, vertrauensunabhängiger Erstellung von Coins.
```

Comment:
```suggestion
Ring-CT führt eine verbesserte Version von @Ringsignaturen - übersetzt "Eine mehrschichtige, verlinkbare, spontane, anonyme Gruppensignatur" genannt - ein. Diese ermöglichen verborgene Beträge und Transaktionsursprünge und -ziele mit akzeptabler Effizienz und verifizierbarer, vertrauensunabhängiger Erstellung von Coins.
```

I have some doubts about "Eine mehrschichtige, verlinkbare, spontane, anonyme Gruppensignatur".. I think there's a paper titled "Multilayered Linkable Spontaneous Anonymous Group (MLSAG) signature scheme".. if people want to read up on that, they'll need the English version or they won't find anything.

Reply:
True. I'll change it back to the English version, but with the translation in brackets.
Like this: (?)
>Ring-CT führt eine verbesserte Version von @Ringsignaturen - "A Multi-layered Linkable Spontaneous Anonymous Group signature" (zu deutsch "Eine mehrschichtige, verlinkbare, spontane, anonyme Gruppensignatur") genannt - ein.

Reply:
+1